packages/@lwc/engine-core/tsconfig.json

```diff
@@ -2,7 +2,9 @@
     "extends": "../../../tsconfig.json",
 
     "compilerOptions": {
-        "outDir": "types",
+        "sourceMap": false,
+        "outDir": "dist",
+        "declarationDir": "types",
```

Comment:
It would be great to normalize across the repo by moving the configuration at the root.

Reply:
The problem is that some server packages (like `@lwc/compiler`) have different configs. For instance, they want `"sourceMap": true, "outDir": "dist/commonjs", "declarationDir": "dist/types"`. I don't think we can make this cleaner unless we do a breaking change where the out/declaration directories change for some packages.

We could, however, ship sourcemaps for every package. But we're not currently doing this, and I didn't want to change the `dist` output with this PR.